Wrappers/Python/test/test_DataContainer.py

```diff
@@ -88,6 +74,88 @@ def test_ndim(self):
         self.assertEqual(x_np.ndim, x_cil.ndim)
         self.assertEqual(3, x_cil.ndim)
 
+    def test_DataContainer_equal(self):
+        array = np.linspace(-1, 1, 32, dtype=np.float32).reshape(4, 8)
+        data = DataContainer(array)
+        data1 = data.copy()
+
+        # Check two identical DataContainers are equal
+        self.assertTrue((data == data1).all())
+        # Check it works for comparing DataContainer with numpy array
+        self.assertTrue((data == data1.as_array()).all())
+
+        # # Check two different DataContainers are not equal
+        data1 += 1
+        self.assertFalse((data ==  data1).all())
```

Comment:
as above

Reply:
ther is no assert array not equal so what would you suggest here?